docs/tutorial.md

```diff
@@ -146,35 +159,97 @@ More on [`bot.craft()`](http://mineflayer.prismarine.js.org/#/api?id=botcraftrec
 
 In this example we create bot that eat / consume held item if hungry.
 Using
-[`health`](http://mineflayer.prismarine.js.org/#/api?id=health) event of the bot to listen on any changes in health or food points.
-[`bot.consume()`](http://mineflayer.prismarine.js.org/#/api?id=botconsumecallback) method to eat.
-[`bot.food`](http://mineflayer.prismarine.js.org/#/api?id=botfood) property to check food points.
-`bot.heldItem` property to get the name of the item held by the bot. `bot.heldItem` is an instance of [`Item`](https://github.com/PrismarineJS/prismarine-item/blob/master/README.md) or `null` if no item held.
+[`health`](http://mineflayer.prismarine.js.org/#/api?id=health) - event that listens for changes in health or food points.
+[`bot.consume()`](http://mineflayer.prismarine.js.org/#/api?id=botconsumecallback) - method that makes bot eat currently held item.
+[`bot.food`](http://mineflayer.prismarine.js.org/#/api?id=botfood) - property to check food points.
+`bot.heldItem` property that returns [`Item`](https://github.com/PrismarineJS/prismarine-item/blob/master/README.md) - held by the bot or `null`.
```

Comment:
item held by bot, remove `-` and add it after `bot.heldItem`. Also link `bot.heldItem`

Reply:
Okay.
I can't find `bot.heldItem` in [the Api page](http://mineflayer.prismarine.js.org/#/api?id=botupdatehelditem).